mulighetsrommet-arena-ords-proxy/src/test/kotlin/no/nav/mulighetsrommet/arena_ords_proxy/ApplicationTest.kt

```diff
@@ -0,0 +1,17 @@
+package no.nav.mulighetsrommet.arena_ords_proxy
+
+import io.ktor.http.*
+import io.ktor.client.request.*
+import io.ktor.client.statement.*
+import kotlin.test.*
+import io.ktor.server.testing.*
+
+class ApplicationTest {
```

Comment:
Kan evt. implementeres med kotest, siden dette brukes andre steder i prosjektet.

Reply:
Det er planen. Dette er kun den default testen som følger med. Prøver heller å separere oppgavene og ikke gjør alt for mye i en giga PR